routing_model/index.html

```diff
@@ -1137,7 +1282,13 @@ <h2>Successfully relocate a node from source to destination section</h2>
     participant Node as Relocating Node
     participant Dst as Destination Section
 
-    Src->>+Dst: Routing RPC: ExpectCandidate
+
+    loop FindDestination
+      Src->>+Dst: Routing RPC: ExpectCandidate
+      opt Refuse
```

Comment:
The title of this section should loose the "successfully"

Reply:
Why, if the node is rejected, the flow is different?

Reply:
Because the diagram now deals with the successful and the unsuccessful situations. We can deal with it in a later PR, though